### PR TITLE
Remove overwhelming text

### PR DESCRIPTION
**Pull request by baristaGeek**
## Description
We're showing way too much text and people don't need instructions every time they open the sidebar. This PR removes such text that users describe as overwhelming. 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)  
- [ ] New feature (non-breaking change which adds functionality)  
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)  
- [ ] Documentation update  
- [x] Chore: cleanup/renaming, etc  
- [ ] RFC

### CODE REVIEW

File: media/utils/setLoading.js [hunk 1]
```diff
@@ -16,12 +16,6 @@ function setError() {
     </div>
     `);
   addActionButtons();
-  $("#ghHolder").append(
```
Comment (EstebanDalelR) [inline media/utils/setLoading.js:19]: So new users won't see it either?
How do they discover usage patterns?

Comment (baristaGeek) [reply media/utils/setLoading.js:19]: Actually, I was also thinking about this. The extension walktrhough is an answer, isn't it?

Comment (EstebanDalelR) [reply media/utils/setLoading.js:19]: I personally have never used that walk through, or read the changelog that pops up.

Comment (EstebanDalelR) [reply media/utils/setLoading.js:19]: I'd prefer to leave it in the Initial HTML, and when reloading, remove it

Comment (baristaGeek) [reply media/utils/setLoading.js:19]: Gotcha, good idea. Let me do the changes.

**Pull request closed without merging**